src/selectors_vm/stack.rs

```diff
@@ -124,10 +124,10 @@ impl TypedChildCounterMap {
                     Some(next) => {
                         v.current = next;
                     },
-                    None => return false
+                    None => return true
                 }
             }
-            true
+            false
         }).for_each(drop);
```

Comment:
This could be `.retain()` instead, and then you could switch from an external `hashbrown` dependency to the standard library.

Reply:
Switching to `retain` is a good idea for simplicity.  However, this code still can't use the standard library HashMap on stable Rust, because it also uses uses `raw_entry_mut`.